libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -133,33 +136,51 @@ describe("Drilling", () => {
             const assertInsightValues = ["69.6%", "68.22%", "71.1%", "30.4%", "31.78%", "28.9%"];
 
             Navigation.visit("dashboard/insight");
+            api.setUpDrillDownAttribute(DISPLAYFORM_PRODUCT, YEAR_CLOSE);
```

Comment:
Because the insight only has 1 drillable attribute, so we have to set up drill down for the second attribute if we want to check chart has two drillable attributes.